CHANGES.md

```diff
@@ -16,6 +16,12 @@
 ------------------------------------------------------------------------------------------------------------------
 
 ## Changes
+- Do not expose additional ports:
+  - Docker compose no longer exposes any container ports outside the default network except for ports 80 and 443 from 
+    the proxy container. This ensures that ports that are not intended for external access are not exposed to the wider 
+    internet even if firewall rules are not set correctly.
```

Comment:
Add a note that `node-exporter` bind on host networking so its port is also exposed.